storage/local/persistence.go

```diff
@@ -932,8 +932,9 @@ func (p *persistence) dropAndPersistChunks(
 	chunkIndexToStartSeek := 0
 	if p.minShrinkRatio < 1 {
 		chunkIndexToStartSeek = int(math.Floor(float64(totalChunks) * p.minShrinkRatio))
```

Comment:
Yes, that is fine. We need to calculate the shrinking ratio for the _resulting_ file. However, if that leads to starting the search outside of the file, we need to check the latest chunk in the file to find out if we can delete the file completely (and create a new one) or alternatively leave everything in place.